mmv1/products/pubsub/Subscription.yaml

```diff
@@ -121,8 +121,18 @@ properties:
         required: true
       - !ruby/object:Api::Type::Boolean
         name: 'useTopicSchema'
+        conflicts:
+          - use_table_schema
         description: |
           When true, use the topic's schema as the columns to write to in BigQuery, if it exists.
+          Only one of useTopicSchema and useTableSchema can be set.
+      - !ruby/object:Api::Type::Boolean
+        name: 'useTableSchema'
```

Comment:
I don't see this in the [published API docs](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions#bigqueryconfig), just want to double-check that it's publicly available & GA?

Reply:
> I don't see this in the [published API docs](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions#bigqueryconfig), just want to double-check that it's publicly available & GA?

Yes, this is GA. The doc update is coming on Monday, which is when it will start to roll out in the UI as well.